com.stencyl.Game.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.stencyl.Game.desktop</id>
+  <name>Stencyl</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>Create games without code</summary>
+  <description>
+    <p>
+	    Stencyl is the easiest way to create games for mobile, web and desktop without code.
```

Comment:
Haha, from one extreme to the other, from no indentation to an overindentation. :P

Consider not using tabs here, then.